packages/react/src/components/contact-form/ContactFormDetails.tsx

```diff
@@ -6,14 +6,15 @@ import From, { Item as ItemForm, GroupItem, ColCountByScreen } from 'devextreme-
 import SelectBox from 'devextreme-react/select-box';
 import Button from 'devextreme-react/button';
 import TextBox from 'devextreme-react/text-box';
-import { PatternRule, EmailRule } from 'devextreme-react/validator';
+import Validator, { PatternRule, EmailRule } from 'devextreme-react/validator';
 
 import { FormPhoto } from '../form-photo/FormPhoto';
 import { ContactStatus } from '../contact-status/ContactStatus';
 import { FormTextbox } from '../form-textbox/FormTextbox';
 
 import { Contact } from '../../shared/types/crm-contact';
 import { CONTACT_STATUS_LIST } from '../../shared/constants';
+import { NumberBox } from 'devextreme-react';
```

Comment:
After build in console appears errors with this record. There is should be:
import NumberBox from 'devextreme-react/number-box;
And move it to top, after `Validator`, to make it more consistently